cups/tasks/main.yml

```diff
@@ -9,6 +9,12 @@
     src: data/{{ job_sheet }}.pdf
     dest: /usr/share/cups/data/
   when: job_sheet != "none"
+- name: check printers.conf for "DeviceURI ipp://printsvr:631/classes/contest"
+  command: grep -Fxq "DeviceURI ipp://printsvr:631/classes/contest" /etc/cups/printers.conf
```

Comment:
I think the template is generated on the master node and first checking if we have the correct values is overkill. So I'm not sure if the extra fact is needed?

Reply:
without this  check each run would replace the printers.conf,  as it changes slightly when cups uses it.

Reply:
Shouldn't we turn off cups, place the file, and then start cups again? Because I think the restart will actually overwrite your file and than load the config which was running.

Reply:
hmm, maybe, but this seems to work in my testing.

Reply:
Ok, I'll see how we did it in another contest if there is a clean way to do this.